composer.json

```diff
@@ -7,7 +7,7 @@
   "type": "flarum-extension",
   "license": "MIT",
   "require": {
-    "flarum/core": ">=0.1.0-beta.12 <0.1.0-beta.14"
+    "flarum/core": "^0.1.0-beta.14"
```

Comment:
Since this package is already used lower-plus-upper limits, I'd suggest keeping it that way.

If we tag a single release with `^`, then it will be installable on all future Flarum betas even if we decide to revert to lower-plus-upper limits later.

Reply:
We use `^` in most FoF extensions, and I think we should stick to that instead of having different constraints.